cpp/modmesh/view/RCameraController.hpp

```diff
@@ -45,12 +45,34 @@
 namespace modmesh
 {
 
+struct CameraInputState
+{
+    float rxAxisValue = 0;
+    float ryAxisValue = 0;
+    float txAxisValue = 0;
+    float tyAxisValue = 0;
+    float tzAxisValue = 0;
+
+    bool leftMouseButtonActive = false;
+    bool middleMouseButtonActive = false;
+    bool rightMouseButtonActive = false;
+
+    bool altKeyActive = false;
+    bool shiftKeyActive = false;
+};
```

Comment:
Same as `Qt3DExtras::QAbstractCameraController::InputState` but with default values. Need default values when setting values in menu [movement items](https://github.com/solvcon/modmesh/pull/408/files#diff-ea4fe5a63732fb5e650cfceb91cfe707209509f69fa3a50533522e0892aa3014R205).